op-test

```diff
@@ -644,3 +644,18 @@ except Exception as e:
     optestlog.error("Exit unexpectedly with Exception={}".format(e))
     OpTestConfiguration.conf.util.cleanup()
     exit(-1)
+finally:
+    # Create a softlink to `latest` test results
+    output = OpTestConfiguration.conf.logdir
+    if not os.path.exists(output):
+        optestlog.error("Test result folder not found")
+        exit(-1)
```

Comment:
It is good to use `sys.exit(-1)`

Reply:
do not see much difference, can change it though..